Add the project description

t-bello7 opened the pull request:
Implemented the following changes in this pull request
- Add project description to project 
- Update readme file

t-bello7 committed:
build: update the readme page

smunozmo reviewed (changes requested):
Hi @t-bello7, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


## Required Changes ♻️

Your capstone is looking great, well done! Let's make a few changes to achieve the styling guidelines. 

## Main page:

First the top bar:

- [x] The dark top bar is missing, it would be great to add it. Also, I would suggest aligning the link to the right as the template shows. Finally, it would be great to add the orange color to the active link.

Template:
![Captura de pantalla 2022-05-27 a la(s) 2 16 34 p  m](https://user-images.githubusercontent.com/83717285/170777153-38e69906-8b40-41db-987a-5a0f69464e9e.png)

Your design:
![Captura de pantalla 2022-05-27 a la(s) 2 16 29 p  m](https://user-images.githubusercontent.com/83717285/170777121-c775feb4-451b-41b2-a6c0-8244649b5868.png)

- [x] Your main title is almost there! However, the title and subtitle are using the same size, let's make "Formula One Page" bigger and add some event-related terms like "... show 2022" as in the original template:

Template:

![Captura de pantalla 2022-05-27 a la(s) 2 12 29 p  m](https://user-images.githubusercontent.com/83717285/170777458-88427407-3d42-4282-b2f3-adeb2abe3386.png)

Your design:
![Captura de pantalla 2022-05-27 a la(s) 2 12 14 p  m](https://user-images.githubusercontent.com/83717285/170777488-79866ea0-f919-4a5d-9244-ad68b411d166.png)

- [x] Great job creating the Main Program section! Let's add the missing horizontal orange line below the title and it would be great to follow the design guidelines for the `see the whole program` button. Finally, the description text is really difficult to read because there's no contrast between the background and the font color:

Template:
![Captura de pantalla 2022-05-27 a la(s) 2 12 58 p  m](https://user-images.githubusercontent.com/83717285/170777780-ec8f075b-7747-41a5-9d93-6e36d654527b.png)

Your design:
![Captura de pantalla 2022-05-27 a la(s) 2 12 44 p  m](https://user-images.githubusercontent.com/83717285/170777737-74147b80-0650-41da-954b-fa88f285c481.png)

- [x] Speakers section is looking good! However, there should be 6 cards, right now there are only 4. Let's add the missing horizontal line below the title. Please add some space to the sides and vertically between the images like in the template:

Template:
![Captura de pantalla 2022-05-27 a la(s) 2 13 20 p  m](https://user-images.githubusercontent.com/83717285/170778591-06eb683e-c150-4246-b243-8e0112005a19.png)

Your design:
![Captura de pantalla 2022-05-27 a la(s) 2 13 11 p  m](https://user-images.githubusercontent.com/83717285/170778663-54974df3-7491-46b2-b3c0-22638a2ea25c.png)

- [x] Partners section is missing on the main page, please add it for review as this is a requirement.

- [x] Footer is missing on the main page, please add it for review as this is a requirement.

## About page:

- [x] You are almost there! Let's remove the empty space below the first section and add an image background as in the template:

Template:
![Captura de pantalla 2022-05-27 a la(s) 2 16 18 p  m](https://user-images.githubusercontent.com/83717285/170779725-9e79685b-2b2c-43b3-bc49-5bb51383e61e.png)

Your design:
![Captura de pantalla 2022-05-27 a la(s) 2 16 11 p  m](https://user-images.githubusercontent.com/83717285/170779770-6c575c45-9cec-4d3d-908a-90bad504ecb8.png)

- [x] Championship winners section is looking good! I would suggest adding some vertical space between the elements and adding the missing orange horizontal line below the title as in the template. Finally, it would be great to add more height to the images:

Template:
![Captura de pantalla 2022-05-27 a la(s) 2 17 28 p  m](https://user-images.githubusercontent.com/83717285/170780150-6ae4773e-5c40-48e5-ba23-01eb0a35ee9b.png)

Your design:
![Captura de pantalla 2022-05-27 a la(s) 2 16 49 p  m](https://user-images.githubusercontent.com/83717285/170780175-005b1d01-fb15-4b84-940c-45fd6f1a6f6f.png)

- [x] Partner section in the About page is been displayed in the desktop version, it would better to show it only in the mobile version as the template shows:

Template:
![Captura de pantalla 2022-05-27 a la(s) 2 53 08 p  m](https://user-images.githubusercontent.com/83717285/170780970-c7603fa5-b52a-48eb-a337-d1cb85a652a3.png)

Your design:
![Captura de pantalla 2022-05-27 a la(s) 2 56 18 p  m](https://user-images.githubusercontent.com/83717285/170781215-a07d474c-e81f-424b-a674-3f0b449a2e8b.png)


- [x] Footer is almost ready! Let's add vertical space as in the template. Keep in mind that the About footer has a dark background in the desktop version and a light one in the mobile version. Please add the correct background color.

Template:
![Captura de pantalla 2022-05-27 a la(s) 2 17 45 p  m](https://user-images.githubusercontent.com/83717285/170780595-61c35632-a08f-477c-90b0-dce950d07170.png)

Your design:
![Captura de pantalla 2022-05-27 a la(s) 2 17 39 p  m](https://user-images.githubusercontent.com/83717285/170780686-75b95d60-0b78-446a-81f7-c7af6da39eb5.png)



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

<img src="https://media1.giphy.com/media/z5cNqbFbMTDA4/giphy.gif"/>

Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

t-bello7 committed:
Fix code review isssues
-change font-size of h1
-add hr to about page
-add top-bar
-change main program overlay background

t-bello7 committed:
Fix code review isssues
-change font-size of h1
-add hr to about page
-add top-bar
-change main program overlay background

t-bello7 committed:
fix merge issues

Ismailco commented on index.html:
```diff
@@ -56,7 +63,7 @@ <h2 class="bold">NEXT RACING EVENT - Sun, 29 May, 14:00</h2>
     </header>
     <main class="container main text-light">
-        <h2 class="center mg-1"> Main Program </h2>
+        <h2 class="center mg-1 main-header"> Main Program </h2>
```
- [ ] [OPTIONAL] Please consider changing the background to the one that was used in the original design or a closer one with the same color
- [ ] [OPTIONAL] Kindly use the same colors as the original design for the program items images and add a hover effect that adds a border to the program items (squares)
- [ ] Kindly center the element and use the propper ```padding```/```margin``` so your design is identical to the given one

|Your design|CC design|
|-|-|
|![Screenshot_20220528_055610](https://user-images.githubusercontent.com/25227022/170810792-4949e796-ef63-429b-8e97-ed9762c6d911.png)|![Screenshot_20220528_060048](https://user-images.githubusercontent.com/25227022/170810805-23dd30d7-4031-4ebc-88a6-056852ea5032.png)|

Ismailco commented on about.html:
```diff
@@ -30,32 +37,33 @@
-                    <li><a href="#">Join</a></li>
+            <div class="navbar flex">
+                <ul class="flex nav">
+                    <li><a class="link" href="./about.html">About</a></li>
```
- [ ] [OPTIONAL] Kindly add a link to the home page from the about page, so the user can navigate easily between the pages

t-bello7 replied:
The requirements for the project required the logo to be linked to the home page.

Ismailco commented on about.html:
```diff
@@ -30,32 +37,33 @@
-        <button class="btn-campaign" type="button"> CC Campaign    </button>
     </nav>
       
     <h1 class="mg-2 center">
```
- [ ] The background dos not respect the accessibility rules and is not the same as the provided design. please consider changing the background image or adding an overlay with opacity on top of the current image, this will make the design closer to the CC design

|Your design|CC design|
|-|-|
|![Screenshot_20220528_060409](https://user-images.githubusercontent.com/25227022/170810982-f4aa0fd7-ad06-4285-8186-b2a38d4298b0.png)|![Screenshot_20220528_060725](https://user-images.githubusercontent.com/25227022/170811002-f8e4949a-ddfc-4f9a-a84b-a57d6ac7e855.png)|

Ismailco commented on index.html:
```diff
@@ -84,10 +91,26 @@ <h2 class="center mg-1"> Main Program </h2>
     </main>
     <section class="container grid flow ff-lato drivers">
-        <h2 class="center drivers-header">Drivers</h2>
+        <h2 class="center drivers-header mg-2">Drivers</h2>
```
- [ ] [OPTIONAL] This section should be generated dynamically, please use JavaScript to generate this HTML code. when creating the JavaScript code kindy consider:

    - The list of speakers used to generate the HTML is assigned to a variable as an array or object and holds their specific data 
    e.g.
   ```
   {
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
  }
   ```
   - There is a function that iterates through the speaker's list on window load (DOM load) and creates HTML elements with the data from each object
   - The dynamic content is attached to the static HTML by selecting the target parent element and adding the generated HTML with the DOM's `appendChild()` or `innerHTML()`

Ismailco commented on about.html:
```diff
@@ -10,7 +10,14 @@
 <body class="ff-lato">
-  <header class="container bg-light header-about">
+  <header class="container header header-about">
+      <div class="topbar text-light bg-dark">
```
- [ ] [OPTIONAL] You can use ```section``` and ```article``` instead of ```<div>``` tag, for more semantic HTML and clean code

Ismailco commented on assets/css/style.css:
```diff
@@ -3,35 +3,64 @@
   background-size: cover;
+  padding-bottom: 2rem;
+}
+
```
- [ ] [OPTIONAL] The pages should look almost identical to the original design, outside the content differences, please use the exact same colors, for text, backgrounds, borders, etc, as the original design
   - Basic color swatch: ```#272A31```, ```#EC5242```, ```#D3D3D3```

Ismailco commented on assets/js/main.js:
```diff
@@ -1,92 +1,11 @@
 const menuBtn = document.querySelector('.btn-menu');
 const closeBtn = document.querySelector('.btn-close');
-const mediaQuery = window.matchMedia('(min-width: 768px)');
 const menu = document.querySelector('.menu');
```
- [ ] [OPTIONAL] The mobile menu is not taking up the whole space, you can give it a width of 100vw

![Screenshot_20220528_063159](https://user-images.githubusercontent.com/25227022/170811625-754fd5d3-631e-42a3-a028-b786fccd52bb.png)

Ismailco reviewed (changes requested):
### Hi @t-bello7,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

t-bello7 committed:
refc: change drivers element to render dynamically

- change home page background for more accessibility

t-bello7 commented:
@Ismailco Thank you for the optional reviews. Motivated me to go a step further in thinking about the accessibility of the website.
I have made the required changes.

CollinsTatang reviewed (approved):
Hi @t-bello7, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

![image](https://user-images.githubusercontent.com/55596970/158098698-83b2f72b-ec7f-42a8-92dd-1814eb6c8a62.png)

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

t-bello7 committed:
fix: add check for drivercontainer

t-bello7 commented:
Thank you @CollinsTatang
